CHANGES.md

```diff
@@ -3,6 +3,9 @@
 ## [Unreleased](https://github.com/crim-ca/stac-populator) (latest)
 
 <!-- insert list items of new changes here -->
+- Adding ability to add collection level assets
+- Adding collection assets to `CMIP6_UofT`
+- Adding an end date to `CMIP6_UofT`'s temporal extent for better rendering in STAC Browser
```

Comment:
The "until present" is not ideal because it's not true. The final date for CMIP6 data is known, the maximum a simulation in the archive goes up to is year 2500. When I originally had "null" I was expecting something like STAC browser to show an undetermined/open end date, not something wrong like "until present".

Reply:
If this is a specific date imposed by CMIP6, then it is fine. I'm not familiar with that granular level of details about it.